src/main/frontend/util.cljc

```diff
@@ -142,28 +142,10 @@
      (p/do!
```

Comment:
There may be some unused fn in this that are useful for the repl. If so, let me know which to keep